notifications.yaml

```diff
@@ -0,0 +1,268 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Notification API
+  description: Notification API.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/messages/{id}':
+    parameters:
+      - name: id
+        in: path
+        type: string
+        required: true
+    get:
+      operationId: getMessage
+      summary: Get Message
+      description: Returns a previously submitted message.
+      responses:
+        '200':
+          description: Message found.
+          schema:
+            $ref: '#/definitions/AcceptedMessage'
+        '404':
+          description: No message found.
+  '/users/{fiscal_number}/messages':
+    parameters:
+      - name: fiscal_number
+        in: path
+        type: string
+        required: true
+        description: The fiscal number of the user.
+        pattern: >-
+          [A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]
+        maxLength: 16
+        minLength: 1
```

Comment:
why minLength = 1? fiscal code [is always 16 characters](https://it.wikipedia.org/wiki/Codice_fiscale#Generazione_del_codice_fiscale) for a person